src/app/Plans/components/Wizard/SelectVMs.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import { Pagination } from '@patternfly/react-core';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  sortable,
+  compoundExpand,
+  classNames as classNamesTransform,
+  ICell,
+  IRow,
+  wrappable,
+} from '@patternfly/react-table';
+import tableStyles from '@patternfly/react-styles/css/components/Table/table';
+
+import { IVMs } from '../../types';
+import { useSortState, usePaginationState, useSelectionState } from '@app/common/hooks';
+import { StatusIcon, StatusType } from '@konveyor/lib-ui';
+
+interface ISelectVMsProps {
+  VMs: IVMs[];
```

Comment:
Is the the naming pattern we would like to settle on for VMs? This seems like a tricky one with it pertaining to a plural abbreviation. Anyone have any opinions about how to name stuff like this?

Reply:
I agree, the interface represents one VM, it should be called `IVM` so that `IVM[]` represents an array of VMs.

Reply:
Also, and this is more of a nitpick, prop names shouldn't be capitalized since that is reserved for components and types. Let's call this `vms`.

Reply:
@ibolton336, @mturley, IVMs was a bad choice.